src/AppInstallerCLIE2ETests/UninstallCommand.cs

```diff
@@ -8,23 +8,101 @@ namespace AppInstallerCLIE2ETests
 
     public class UninstallCommand : BaseCommand
     {
-        private const string UninstallTestExeInstalledFile = @"TestExeUninstalled.txt";
+        private const string CustomProductCode = "{f08fc03c-0b7e-4fca-9b3c-3a384d18a9f3}";
+        private const string UninstallTestExeUninstalledFile = "TestExeUninstalled.txt";
+        private const string UninstallTestMsiInstalledFile = "AppInstallerTestExeInstaller.exe";

```

Comment:
>UninstallTestMsiInstalledFile = "AppInstallerTestExeInstaller.exe"; [](start = 29, length = 67)

this line does not look correct

Reply:
ah, I see now. Please add a comment on why it's this because on first sight, it's confusing

---
In reply to: [543043894](https://github.com/microsoft/winget-cli/pull/659#discussion_r543043894) [](ancestors = 543043894)